src/commands/setup.md

```diff
@@ -73,6 +73,11 @@ The followings shall be created
    1. A Lifecycle pipeline.
    1. A Build pipeline.
 
+## Pre-requisite
+
+azure cli needs to be installed so that pull request can be automatically
+approved. type `az version` to check if you have version 2.0.x installed.
+
```

Comment:
We also need the az extension `azure-devops`, maybe the docs should mention that?

Reply:
yes. thanks

Reply:
@samiyaakhtar Do you have any further feedback?